### PR TITLE
1102 - Issues with errors

### PR DESCRIPTION
**Pull request by aaroncarroll82**
* Remove extra error_summary class. 
* Order errors correctly on refer case screen. 
* Enable possibility to click on errors in summary and jump to errors in the embedded parts of radio group
* Make error descriptions match in note field and error summary
* Make error description match for cancel reason and error summary
* Make sure note field is constructed in accessible way on the crit evidence pages

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala [hunk 1]
```diff
@@ -41,7 +41,7 @@ class SuspendCaseController @Inject()(verify: RequestActions,
 
   override protected val config: AppConfig = appConfig
   override protected val caseService: CasesService = casesService
-  private val form: Form[String] = AddNoteForm.form
+  private val form: Form[String] = AddNoteForm.getForm("error.empty.suspend.note")
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala]: Seems a bit of a strange thing to be passing into a `getForm` method? maybe just pass in `suspend` and the `error.empty.{x}.note` key is generated inside the form?

File: app/uk/gov/hmrc/tariffclassificationfrontend/forms/mappings/FormMappings.scala [hunk 2]
```diff
@@ -46,4 +46,17 @@ object FormMappings {
         Map(key -> value)
     })
 
+  def oneOf(errorKey: String = "error.required", enumeration: Enumeration): FieldMapping[String] =
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/forms/mappings/FormMappings.scala]: Is this unit tested?

Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/forms/mappings/FormMappings.scala]: Not directly (none of the methods in FormMappings are), but it is as part of the form that uses it. Couldn't hurt to add some unit tests for the methods in here too I guess

Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/forms/mappings/FormMappings.scala]: Added unit tests for all 3 methods now.

File: app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala (hunk 1 above)
Comment (aaroncarroll82) [reply app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala]: updated as suggested

**Pull request merged**